packages/core/package.json

```diff
@@ -7,7 +7,10 @@
 	"authors": [
 		"The Preact Authors (https://github.com/preactjs/signals/contributors)"
 	],
-	"repository": "preactjs/signals",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/preactjs/signals"
+	},
```

Comment:
From what I understand, in order for NPM to work with relative links in our ReadMes, we need to update the specifier here.

[Source](https://stackoverflow.com/a/68738752/15388164)